### PR TITLE
Added some important methods to  pull_request in Github and created Gitlab 

### PR DESCRIPTION
**Pull request by shivvamm**
this is for the issue #223

### CODE REVIEW

File: src/scrape_up/github/pull_request.py [hunk 1]
```diff
@@ -131,3 +131,173 @@ def reviewers(self):
...
+    return f"https://github.com/{self.username}/{self.repository}/pull/{self.pr_number}/commits" if self.pr_number else None
+
+    def diff_url(self):
+    """
+    Get the URL to view the diff of changes in a pull request
+    """
+    return f"https://github.com/{self.username}/{self.repository}/pull/{self.pr_number}.diff" if self.pr_number else None
+
+    def html_url(self):
+    """
```
Comment (nikhil25803) [inline src/scrape_up/github/pull_request.py:300]: Hey @shivvamm, please follow the code structure of the project while returning the output as shown in the picture below.
image

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/pull_request.py (hunk 1 above)
Comment (shivvamm) [reply src/scrape_up/github/pull_request.py:300]: Okay sure I'll update it now

Comment (shivvamm) [reply src/scrape_up/github/pull_request.py]: Done I fixed it

File: documentation.md [hunk 2]
```diff
@@ -205,3 +205,9 @@ following = user.following() #user var taken from above
 print(following)
 ```
 ---
+
+## Gitlab
+
+```python
+from scrape_up import gitlab
```
Comment (nikhil25803) [inline documentation.md:212]: Do not just mention the class but also define all the methods used as shown in the example below
image

File: src/scrape_up/gitlab/users.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        for repo_element in repo_elements:
+            name = repo_element.find("h3", class_="project-title").text.strip()
+            description = repo_element.find("p", class_="project-description").text.strip()
+            repositories.append({
+                "name": name,
+                "description": description
+            })
+        
+        # Return the list of repositories
+        return repositories
```
Comment (nikhil25803) [inline src/scrape_up/gitlab/users.py:77]: Same as others, the return method is not implemented as per the project structure, also no try-catch is used as well.
Please look into it

File: src/scrape_up/gitlab/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+from scrape_up.gitlab.users import Users
+from scrape_up.gitlab.project import Project
```
Comment (nikhil25803) [inline src/scrape_up/gitlab/__init__.py:2]: Mention all the Class of the `scrape_up.gitlab`. 
image

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Hey @shivvamm 
A lot of changes are required from your side. Take your time and make your changes correctly.

**Comment by nikhil25803** [comment]
Hey @shivvamm, any update?

**Comment by shivvamm** [comment]
Yes I'm working on it just bit busy

**Comment by nikhil25803** [comment]
No issue @shivvamm, take your time.

**Pull request closed without merging**